source/api/commands/as.md

```diff
@@ -1,9 +1,8 @@
 ---
 title: as
-
```

Comment:
I noticed that this extra newline was being removed in recent PRs so I duly removed it here too for consistency.